src/sections/Meshery/Meshery-integrations/Honeycomb/Honeycomb.js

```diff
@@ -4,6 +4,8 @@ import { HoneycombGrid } from "./Honeycomb.style";
 const Honeycomb = (props) => {
   const { items, renderItem } = props;
   const [height, setHeight] = useState(0);
+  const [heightMobile, setHeightmobile]= useState(0);
```

Comment:
Why are these extra height states required?

Reply:
Because when you see all the integration then in bottom the honeycomb structure is overlapping with other next component . and also the height is depend on honey comb size that why we need separate height for mobile screen .